doc/source/examples/unresolved-cfd-dem/dense-pneumatic-conveying/dense-pneumatic-conveying.rst

```diff
@@ -0,0 +1,636 @@
+   end
+
+Simulation Control
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The simulation lasts 5 seconds and the CFD time step is 5e-4 seconds.
+
+.. code-block:: text
+
+    subsection simulation control
+      set method               = bdf1
+      set output name          = cfd_dem
+      set output frequency     = 10
+      set time end             = 5
+      set time step            = 5e-4
+      set output path          = ./output/
+    end
+
+Physical Properties
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The physical properties of air are the same as Lavrinec *et al*. [#lavrinec2021]_
+
+.. code-block:: text
+
+    subsection physical properties
+      subsection fluid 0
+        set kinematic viscosity = 1.5e-5
+        set density             = 1.205
+      end
+    end
+
+Boundary Conditions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The boundary condition at the wall of the pipe is a weak function where the Dirichlet condition is weakly imposed as a no-slip condition, since we specify zero velocity. The inlet and the outlet have periodic boundaries. `See here <../../../parameters/cfd/boundary_conditions_cfd.html>`_ for more information about those boundary conditions.
+
+.. code-block:: text
+
+    subsection boundary conditions
+      set number = 2
+      subsection bc 0
+        set id   = 0
+        set type = function weak
+        set beta = 100
+        subsection u
+          set Function expression = 0
+        end
+        subsection v
+          set Function expression = 0
+        end
+        subsection w
+          set Function expression = 0
+        end
+      end
+      subsection bc 1
+        set id                 = 1
+        set type               = periodic
+        set periodic_id        = 2
+        set periodic_direction = 0
+      end
+    end
+
+Flow control
+~~~~~~~~~~~~~~~~~~
+
+Since the simulation has periodic boundary conditions, a correction force is needed to drive the flow to compensate the pressure drop in the pipe. In other to achieve this, we use the `dynamic flow controller <../../../parameters/cfd/dynamic_flow_control.html>`_. Here, we also apply a proportional beta force on particles. The average velocity is set to 3 m/s, this correspond to the average over the entire domain considering the void fraction. The flow controller performs well for CFD simulation, but needs some tuning for CFD-DEM simulation. By default, the controller has a high stiffness and aims to correct the flow in the next time step. However, the carrying of particles by the flow leads to a response time that is not taken into account and results in a oscillation of the velocity of the flow. To avoid this, we use the beta threshold and the alpha relaxation parameter. Here, the beta value will not be updated if the new beta value is within the 5% of the previous value. Also, the correction to apply to the previous beta value is reduced by a factor of 0.25. This way, the velocity of the flow and the particles are more stable.
```

Comment:
```suggestion
Since the simulation has periodic boundary conditions, a correction force is needed to drive the flow to compensate the pressure drop in the pipe. In other to achieve this, we use the `dynamic flow controller <../../../parameters/cfd/dynamic_flow_control.html>`_. Here, we also apply a proportional force (..math: `\beta`) on the particles. The average velocity is set to 3 m/s, this correspond to the average over the entire domain considering the void fraction. The flow controller performs well for CFD simulation, but needs some tuning for CFD-DEM simulation. By default, the controller has a high stiffness and aims to correct the flow in the next time step. However, the transport of particles by the flow leads to a response time that is not taken into account by the density of the gas and results in a oscillation of the velocity of the flow. To avoid this, we use the beta threshold and the alpha relaxation parameter. Here, the beta value will not be updated if the new beta value is within the 5% of the previous value. Also, the correction to apply to the previous beta value is reduced by a factor of 0.25. This way, the velocity of the flow and the particles are more stable.
```

I would use the symbol "beta" everywhere instead of saying beta. It's not like a "beta force" it's just a force that we call beta.